coderd/templateversions.go

```diff
@@ -63,6 +63,36 @@ func (api *API) templateVersion(rw http.ResponseWriter, r *http.Request) {
 	httpapi.Write(ctx, rw, http.StatusOK, convertTemplateVersion(templateVersion, convertProvisionerJob(job), user))
 }
 
+// @Summary Patch template version by ID
+// @ID patch-template-version-by-id
+// @Security CoderSessionToken
+// @Produce json
+// @Tags Templates
+// @Param templateversion path string true "Template version ID" format(uuid)
+// @Param request body codersdk.PatchTemplateVersionRequest true "Patch template version request"
+// @Success 200 {object} codersdk.TemplateVersion
+// @Router /templateversions/{templateversion} [patch]
+func (api *API) patchTemplateVersion(rw http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	templateVersion := httpmw.TemplateVersionParam(r)
+	var params codersdk.PatchTemplateVersionRequest
```

Comment:
Technically, in a patch, you should check if the value is given before updating it. In this code, if `name` is omitted from the request, the template version would receive a blank name, which isn't RESTful.

Reply:
My point makes more sense when there are multiple possible values.

Reply:
Added it to make easier to identify in the near future